docs/0.4/references/api/index.md

```diff
@@ -0,0 +1,9 @@
+# Splinter REST API Reference
+
+This API defines the endpoints for Splinter daemon, `splinterd`. These endpoints
```

Comment:
s/for Splinter daemon/for the Splinter daemon/

Reply:
Fixed.